deploy-manage/security/secure-settings.md

```diff
@@ -260,23 +204,24 @@ There are reloadable secure settings for:
 * [LDAP realm](elasticsearch://reference/elasticsearch/configuration-reference/security-settings.md#ref-ldap-settings)
 * [Remote cluster credentials for the API key based security model](/deploy-manage/remote-clusters/remote-clusters-settings.md#remote-cluster-credentials-setting)
 
-
-::::
-
-:::::
-
-## The {{kib}} keystore
+## {{kib}} secure settings [kibana]
 ```{applies_to}
 deployment:
   self: ga
+  eck: ga
 ```
 
-Some settings are sensitive, and relying on filesystem permissions to protect their values is not sufficient. For this use case, {{kib}} provides a keystore, and the `kibana-keystore` tool to manage the settings in the keystore.
+{{kib}} supports secure settings through its own keystore, similar in nature to the {{es}} keystore. It provides a way to protect sensitive configuration values, such as authentication credentials or encryption keys, by storing them securely outside of `kibana.yml`.
 
 ::::{note}
-* Run all commands as the user who runs {{kib}}.
-* Any valid {{kib}} setting can be stored in the keystore securely. Unsupported, extraneous or invalid settings will cause {{kib}} to fail to start up.
+Unlike with {{es}} keystore, any valid {{kib}} setting can be stored securely in the keystore. However, as with {{es}}, adding invalid, unsupported, or extraneous settings will cause Kibana to fail to start. Always ensure the setting exists and is properly formatted in the [configuration reference](kibana://reference/configuration-reference.md) before adding it to the keystore.
+::::
+
+This section provides examples of {{kib}} secure settings handling using the `kibana-keystore` command-line tool.  
+For ECK deployments, the configuration process is similar to {{es}} and is documented in the [{{kib}} secure settings section for ECK](./k8s-secure-settings.md#k8s-kibana-secure-settings).
 
+::::{important}
+In the following examples, run all commands as the user who runs {{kib}}.
 ::::
 
 ### Create the keystore [creating-keystore]
```

Comment:
same `kibana.keystore` note here

Reply:
I now understand what you meant with the password vs no password. All fixed.
Kibana part was already clear by the way!